pyaedt/modeler/Primitives3DLayout.py

```diff
@@ -616,16 +616,16 @@ def padstacks(self):
         return self._padstacks
 
     @pyaedt_function_handler()
-    def change_net_visibility(self, netlist=None, visible=False):
+    def change_net_visibility(self, netlist=None, visible=True):
```

Comment:
Do we really need to modify the default value.
Indeed, it will modify the method signature and could break some existing users' scripts.